src/javascript/app/common/form_manager.js

```diff
@@ -136,7 +136,11 @@ const FormManager = (() => {
             can_submit  = form.can_submit;
             if (!can_submit) return;
             if (Validation.validate(options.form_selector)) {
-                const req = $.extend({}, options.obj_request, getFormData(options.form_selector));
+                const req = $.extend({},
+                    options.obj_request,
+                    getFormData(options.form_selector),
+                    options.static_values
```

Comment:
how about request_fields or other_request_fields?

Reply:
and whats the difference between static_values and obj_request?

Reply:
You’re right I'll remove `static_values` we can add a nested object in obj_request